polynote-runtime/src/main/scala/polynote/runtime/package.scala

```diff
@@ -0,0 +1,5 @@
+package polynote
+
+package object runtime {
+  type CellRange = (Int, Int) // use this for readability when you have some (start: Int, end: Int) range denoting a selection in a cell.
```

Comment:
Not sure whether there might be a better place to put this.

Reply:
I wonder if we should just make this a `case class` of its own? That way the fields would be nominal instead of ordinal... shouldn't really affect anything else a whole lot. (Not urgent though)

Reply:
good idea, i'll add a TODO

Reply:
TODOne